game/src/main/kotlin/world/gregs/voidps/world/map/lumbridge/LumbridgeFlag.kts

```diff
@@ -0,0 +1,21 @@
+package world.gregs.voidps.world.map.lumbridge
+
+import world.gregs.voidps.engine.entity.character.forceChat
+import world.gregs.voidps.engine.entity.character.player.Player
+import world.gregs.voidps.engine.entity.character.setAnimation
+import world.gregs.voidps.engine.entity.obj.ObjectOption
+import world.gregs.voidps.engine.event.on
+import world.gregs.voidps.engine.queue.softQueue
+
+
+on<ObjectOption>({ operate && target.id == "lumbridge_flag" && option == "Raise" }) { player: Player ->
+    player.setAnimation("lumbridge_flag_raise")
```

Comment:
Make sure to have `arriveDelay()` at the start of Object interactions; it delays the player until they've finished moving/right beside the object before starting to interact.

Reply:
ok iv added it now